File: android/app/src/main/java/com/dongchyeon/simplechatapp/retrofit/Result.java
package com.dongchyeon.simplechatapp.retrofit;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {
    @SerializedName("result")
    @Expose
    private Integer result;

    @SerializedName("imageUri")
    @Expose
    private String imageUri;

    public Integer getResult() { return result; }

    public void setResult(Integer result) { this.result = result; }

    public String getImageUri() { return imageUri; }

    public void setImageUri(String imageUri) { this.imageUri = imageUri; }
}

File: android/app/src/main/java/com/dongchyeon/simplechatapp/data/model/Result.kt
package com.dongchyeon.simplechatapp.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Result(
    @SerializedName("result")
    @Expose
    val result: Int,

    @SerializedName("imageUri")
    @Expose
    val imageUri: String
)